Stripe/STPFormTextField.m

```diff
@@ -206,6 +207,31 @@ - (void)setAttributedText:(NSAttributedString *)attributedText {
     }
 }
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wpartial-availability"
+// accessibilityAttributedValue is only defined on iOS 11 and up, but we
+// implement it immediately below, so we should just ignore the warning.
+- (NSString *)accessibilityValue {
+    return [[self accessibilityAttributedValue] string];
+}
+#pragma clang diagnostic pop
+
+- (NSAttributedString *)accessibilityAttributedValue {
+    NSMutableAttributedString *attributedString = [self.attributedText mutableCopy];
+    #ifdef __IPHONE_13_0
+    if (@available(iOS 13.0, *)) {
+        [attributedString addAttribute:UIAccessibilitySpeechAttributeSpellOut value:@(YES) range:NSMakeRange(0, [attributedString length])];
+    }
+    #endif
+    if (!self.validText) {
+        NSString *invalidData = STPLocalizedString(@"Invalid data.", @"Spoken during VoiceOver when a form field has failed validation.");
+        NSMutableAttributedString *failedString = [[NSMutableAttributedString alloc] initWithString:invalidData attributes:@{UIAccessibilitySpeechAttributePitch: @(0.6)}];
```

Comment:
You can control the pitch, how neat!

Reply:
Safari doesn't do this, but I thought it'd be nice to make it more obvious that this is a system message and not part of the text field's value.